plugin/db/driver.go

```diff
@@ -188,6 +189,9 @@ const (
 	// Migrate is the migration type for MIGRATE.
 	// Used for DDL change including CREATE DATABASE.
 	Migrate MigrationType = "MIGRATE"
+	// SDLMigrate is the migration type for SDL_MIGRATE.
+	// Used for SDL based migration.
+	SDLMigrate MigrationType = "SDL_MIGRATE"
```

Comment:
What's the specific reason that we need to introduce a new migration type. This will complicate code to handle 2 migration type in some places and easy to miss.

https://sourcegraph.com/search?q=context:bytebase+db.Migrate&patternType=regexp

https://sourcegraph.com/search?q=context:bytebase+%22MIGRATE%22&patternType=regexp&case=yes

Reply:
If we do, we also need to update the comment in schema file https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/plugin/db/mysql/mysql_migration_schema.sql?L26&subtree=true

Reply:
We may have to introduce a new type unfortunately because SDL migration is a different mechanism from standard DDL migration just as the difference between DDL and DML. Updated.

Reply:
What are the specific places that we need to differentiate? It's unclear from this PR since I don't see the use of SDLMigrate

Reply:
It’s an introduction but not yet used. If we feel uncertain about the change, I can make it to MIGRATE for now and change it later if needed.

Reply:
Yeah, I would prefer to keep it for MIGRATE for now.

This also reminds us that the SDL is not ready to be released. We probably need to implement more features to see if we do need to introduce a new type.

Reply:
BTW, if we do need to introduce a new type, I would consider naming it MIGRATE_SDL

Reply:
MIGRATE_SDL SG. Yup, SDL isn't quite ready yet, and there are a few more changes needed to bake it, e.g. computing the schema diff shouldn't be part of webhook.go but task executor.